account_invoice_mgmt_dms/models/account_move.py

```diff
@@ -45,9 +46,12 @@ def _compute_state_complete_proceesing(self):
 
     @api.depends('dms_file_ids')
     def _compute_complete_proceesing_id(self):
+        directory_id = self.env.ref('account_invoice_mgmt_dms.dms_directory_puchase_invoice')
         for record in self:
             if record.dms_file_ids.filtered(lambda x: x.complete_proceeding):
                 record.complete_proceesing_id = record.dms_file_ids.filtered(lambda x: x.complete_proceeding)
+            if record.dms_file_ids.filtered(lambda x: x.directory_id == directory_id):
```

Comment:
Please ensure corner case value (should be `record.purchase_invoice_proceesing_id` be unset when no dms directory is found?)

Reply:
This record will always exist, since it is one that we added to the data